docs/_extra/api-reference/hypothesis.yaml

```diff
@@ -1,63 +1,13 @@
 swagger: "2.0"
 info:
   version: 1.0.0
-  title: Hypothesis
+  title: Hypothesis API Reference
   description: |
-    # Hypothesis API documentation
+    # Hypothesis API Reference
 
-    This document details the Hypothesis public HTTP API. It is targeted at
```

Comment:
Here I'm moving the main authorization documentation out of ReDoc (now under "API Reference" in the main table of contents) and into a separate "Authorization" page authored in reST.